rs/embedders/src/wasmtime_embedder/system_api.rs

```diff
@@ -1213,10 +1213,8 @@ pub(crate) fn syscalls<
                         system_api.ic0_call_with_best_effort_response(timeout_seconds)
                     })
                 } else {
-                    let err = HypervisorError::UserContractViolation {
+                    let err = HypervisorError::ToolchainContractViolation {
```

Comment:
Switching this do a toolchain error because the api isn't exposed in the CDK yet. So regular users shouldn't see this error.